src/pudl/metadata/sources.py

```diff
@@ -447,7 +447,9 @@
             "summer-winter generation peaks and system lambda."
         ),
         "field_namespace": "ferc714",
-        "working_partitions": {},  # Data is monolitic, one file with all years.
+        "working_partitions": {
+            "years": sorted(set(range(2021, 2022))),  # Pre-XBRL data is monolithic
+        },
```

Comment:
How does this change affect the ETL of the exising FERC 714 data?

Reply:
Turns out this wasn't an issue because of how the FERC 714 ETL work snow, but we'll need to refactor the data sources + working partitions to deal with multiple differently structured raw source data.